Detector/DetCRD/compact/CRD_o1_v01/CRD_o1_v01-onlyTracker.xml

```diff
@@ -32,7 +32,7 @@
   <include ref="../CRD_common_v01/VXD_v01_01.xml"/>
   <include ref="../CRD_common_v01/FTD_SkewRing_v01_01.xml"/>
   <include ref="../CRD_common_v01/SIT_SimplePixel_v01_01.xml"/>
-  <include ref="../CRD_common_v01/DC_Simple_v01_02.xml"/>
+  <include ref="../CRD_common_v01/DC_Simple_v01_03.xml"/>
```

Comment:
I can't find DC_Simple_v01_03.xml. Could you please double check?

Reply:
Hi @myliu-hub , could you reply Yao's comment?